mkt/tags/models.py

```diff
@@ -37,25 +37,23 @@ def can_reverse(self):
     def get_url_path(self):
         return reverse('tags.detail', args=[self.tag_text])
 
-    def save_tag(self, addon):
+    def _get_m2m_name(self, obj):
+        model = obj._meta.model_name
+        if model == 'webapp':
+            return 'tags'
+        elif model == 'website':
+            return 'keywords'
```

Comment:
You
Can Can trotting by jbbbnhi Bingham B in nbg

by nbgg
 On Jun 13, 2015 1:35 PM, "Rob Hudson" <notifications@github.com> wrote:

> In mkt/tags/models.py
> <https://github.com/mozilla/zamboni/pull/3132#discussion_r32372247>:
>
> > @@ -37,25 +37,23 @@ def can_reverse(self):
> >      def get_url_path(self):
> >          return reverse('tags.detail', args=[self.tag_text])
> >
> > -    def save_tag(self, addon):
> > +    def _get_m2m_name(self, obj):
> > +        model = obj._meta.model_name
> > +        if model == 'webapp':
> > +            return 'tags'
> > +        elif model == 'website':
> > +            return 'keywords'
>
> Curious if there's a smarter way to do this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/zamboni/pull/3132/files#r32372247>.
>